src/components/footer/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { socialMediaIcons } from '../../constants';
+
+import './Footer.scss';
+
+export const Footer = () => {
+    return (
+        <div className={'may-footer'}>
```

Comment:
if you want to pass simple string as prop you can skip using curly brackets 
and it would be `className='may-footer'`